src/components/Home/nft.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
```

Comment:
Can we put all of the boxes in one file? I think that's a bit of an excessive amount of files.